src/scss/includes/search_form.scss

```diff
@@ -154,11 +154,9 @@ input[type="search"] {
   width: 60px;
   flex-shrink: 0;
   text-align: center;
-  margin-right: 14px;
```

Comment:
This rule is here so (on desktop at least), the suggestion labels are vertically aligned with the input content. Let's wait for a more general scope redesign of all suggests to change it.

Reply:
Your screenshots show exactly what I describe:
|Before|After|
|---|---|
|![95210149-7b2a2280-07eb-11eb-86d7-6d0a4129b7fc](https://user-images.githubusercontent.com/243653/95211067-6c903b00-07ec-11eb-91cb-97b3d80de98a.png)|![95210298-a3b21c80-07eb-11eb-8c02-ac32363304a3](https://user-images.githubusercontent.com/243653/95211121-787bfd00-07ec-11eb-8554-fa985fd48bb8.png)|

And yes, this alignment is on purpose :smile: 

My point is you are changing something outside of the scope of directions only, so we should manage it we care, and there is no rush to do it here.

Reply:
Thomas just validated this change even if it applies to both Directions and Top bar.
Let's merge it like that and wait for a future redesign of this panel 👍